### ISSUE TITLE
Screen reader does not announce the tooltip information for the "colors list items" present under "Change color" control.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#203]
### User Experience:
If there is no name is defined for the controls and screen reader does not announce the name and also the tooltip information, then, users will not be able to understand the purpose of the control and users will get impacted to understand the name of the control.

### Repro steps:
- Navigate to the Left nav and select Pages button.
- Navigate to the Pages window. select API Reference option.
- Navigate to the any editable control details and select Add widget.
- Navigate to the Text Block and select it.
- Navigate to the "Text Block" blade and select edit button.
- Navigate to the various controls of the "Text Block" dialog box.
- Verify whether screen reader announcing the color information or not.

### Actual Result:
Name is not defined for the colors list items and screen reader does not announce the tooltip information for the list items.
Observation:
While navigating, screen reader is announcing as 1 of 18 instead of its tooltip information.
While navigating with all screen reader (NARRATOR, JAWS, NVDA)
When NARRATOR focus lands on "1st list item", NARRATOR screen reader is announcing as "1 of 18".
When NVDA focus lands on "1st list item", NVDA screen reader is announcing as "list items 1 of 18".
When JAWS focus lands on "1st list item", JAWS screen reader is announcing as "Colors list 1 of 18".

### Expected Result:
Name should be defined, and screen reader should announce the "Tooltip" information for the "colors list items" present under "Change color" control.
Ex: When screen reader focus lands on "1 list item then it should announce it as "Clear color 1 of 18".

**Comment by azaslonov** [comment]
Fixed in 0.1.540.
